public/locales/en.json

```diff
@@ -188,6 +188,7 @@
     "label_status_pending": "In progress",
     "label_status_running": "In progress",
     "label_status_paused": "Paused",
+    "label_status_not_started_sources": "Scan not run yet",
```

Comment:
optional: run this by ccs/docs

Reply:
Docs suggested "Scan not started", which I now use in fd447c501c5a4852f581ee2e81b41e8adfb3485f